### DISCUSSION TITLE
Potential naming issue

### DISCUSSION THREAD

**Discussion by enslaved2die** [tenacityteam/tenacity-legacy#252] [General]
I like the new name Tenacity but this is already the name of a Python library and could possible cause issues in package managers.
https://github.com/jd/tenacity

**Comment by fossdd** [comment]
Many package managers like the Arch Linux Packages have a prefix if it's a library like `python-tenacity`

**Reply by poyribn** [reply]
Still: Avoiding confusion is better, so we need another name.

**Reply by bitsper2nd** [reply]
What about [Brass?](https://www.merriam-webster.com/dictionary/brass#synonyms) Seems like a [perfect fit](https://en.wikipedia.org/wiki/Brass_instrument).

![image](https://user-images.githubusercontent.com/22452474/125510688-76185e24-4adc-40b2-9ca1-00d6796a9b0f.png)

**Reply by Semisol** [reply]
We have mostly locked in on the name, so...

> Still: Avoiding confusion is better, so we need another name.

I don't think anyone will confuse a Python library with an audio editor.

**Reply by enslaved2die** [reply]
Its not about confusing a Phython library with a audio editor its a about a possible issue with package manage.
but as @fossdd mention most of them use prefixes. maybe our tenacity has to use one too :)

**Reply by poyribn** [reply]
> I don't think anyone will confuse a Python library with an audio editor.

Great news to me:
We Don't need a new name.
Maybe the package name should be `tenacity-audio` to further reduce confusion with `python-tenacity`,
but otherwise the name "tenacity" can stay.

**Reply by mc776** [reply]
As long as someone at 1AM suddenly getting an idea for an audio thing, but not having touched any audio editing since they'd deleted Audacity, can type sudo apt-get install tenacity and hit Tab and figure out what's going on and install the right thing the first time, it's all good.

**Comment by Semisol** [comment]
A Python library for retrying doesn't exactly sound something so easy to confuse with an audio editor.
And the name is sort of locked in so, it will be hard changing.

**Reply by poyribn** [reply]
Great news to me:
We Don't need a new name.
Maybe the package name should be `tenacity-audio` to further reduce confusion with `python-tenacity`,
but otherwise the name "tenacity" can stay.

**Reply by Be-ing** [reply]
`tenacity-audio` would be fine for a package name if packagers choose to do so.

**Reply by fossdd** [reply]
> A Python library for retrying doesn't exactly sound something so easy to confuse with an audio editor.

Sure, but if both packages have the same code name, well that doesn't works.

**Reply by fitojb** [reply]
The mentioned library isn’t packaged in either Debian or Ubuntu (just checked Impish), so `tenacity` has that name freed in those operating systems and their derivatives.

**Comment by mc776** [comment]
Suddenly feel like giving Chromium B.S.U. another try...

**Comment by AnotherFoxGuy** [comment]
According to repology the python package is called `python-tenacity` in almost all repos except on pip, so we don't have any issues with naming 
https://repology.org/project/python:tenacity/versions

**Comment by RAMChYLD** [comment]
I believe this is a non issue. The exact same problem plagues Chromium. No one is going to confuse a python library for an audio editor just like no one is going to confuse a top-down shmup for a web browser.
